src/custom/hooks/useMarkdown.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react'
+
+export default function useMarkdown(file: string) {
+  const [content, setContent] = useState('')
+
+  useEffect(() => {
+    const fetchContent = async () => {
+      await fetch(file)
```

Comment:
I'm surprised we use FETCH for the markdown, I was expecting to load it from the project. Normal import. 
However, I see this can be useful for some occasions if we want to maintain the file from outside.

Reply:
I see you added the markdown in the project. I would suggest to leave this hook, but not strictly need it for this page. 

Also, there's nothing related to Markdown in this hook, it feels is a hook to get the content from a HTTP request. You can call this `useFetch` cause it feels like just a wrapper of fetch